src/include/network/remote/dialogs.rb

```diff
@@ -60,9 +60,17 @@ def RemoteMainDialog
           # RadioButton label
           Left(
             RadioButton(
-              Id(:allow),
-              _("&Allow Remote Administration"),
-              Remote.IsEnabled
+              Id(:allow_with_vncmanager),
+              _("&Allow Remote Administration With Session Management"),
+              Remote.IsEnabled && Remote.UsesVncManager
```

Comment:
`Remote.UsesVncManager` has been renamed away. The bug is hidden even if I run `rake run[remote]` because `IsEnabled` returns false.